Referenced by issue #53713, Python API: user_remap for images seems broken
